### PR TITLE
Replace select and option with sl-select and sl-menu-item

### PR DESCRIPTION
**Pull request by dlaliberte**
Replace select and option with sl-select and sl-menu-item.  Also uses sl-icon.

Also fixes the shoelace base path, because we need to use the check-lg and chevron-right icons.

Here is what it looks like with an opened select box and preselected item;

image

### CODE REVIEW

**Review by jrobbins — APPROVED** [review]
This all looks good.

In our meeting today, I had mentioned that I might deploy this today, but instead I think I will deploy without this change.  The reason is that I'd like to wait for your follow-up PR (or another commit and re-review on this one) that hides the list of choices until after the component JS loads.

**Comment by dlaliberte** [comment]
> In our meeting today, I had mentioned that I might deploy this today, but instead I think I will deploy without this change. The reason is that I'd like to wait for your follow-up PR (or another commit and re-review on this one) that hides the list of choices until after the component JS loads.

That's fine.  I tried the body.loading sl-select selector, but it doesn't seem to be enough. since the shoelace code doesn't convert the web components soon enough.  However, this general problem is addressed in the shoelace docs:
https://shoelace.style/getting-started/usage?id=waiting-for-components-to-load
I'll try a few things.  I think we need to use display:none on the sl-menu-item elements, however, since we don't want them to take up any space.

File: static/sass/forms.scss
Comment (jrobbins) [inline static/sass/forms.scss:148]: This would be more readable with spaces around the `+`.

**Review by jrobbins — COMMENTED** [review]
LGTM after one small change.

File: static/sass/forms.scss
Comment (jrobbins) [reply static/sass/forms.scss:148]: Oh, you already merged it.  No big deal, just a style nit.

**Pull request merged**
